core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2005,7 +2005,7 @@ object Log extends Logging {
     Files.createDirectories(dir.toPath)
     val topicPartition = Log.parseTopicPartitionName(dir)
     val segments = new LogSegments(topicPartition)
-    val leaderEpochCache = Log.maybeCreateLeaderEpochCache(dir, topicPartition, logDirFailureChannel, config.messageFormatVersion.recordVersion)
+    val leaderEpochCache = Log.maybeCreateLeaderEpochCache(dir, topicPartition, logDirFailureChannel, config.messageFormatVersion.recordVersion, this.logIdent)
```

Comment:
Hmm, it seems that `this.logIdent` is always empty?

Reply:
Done. Nice catch. Addressed in cc8353fd9e2475e56488ade8d6f01a772830edc4.